pkg/apis/autoscaling/types.go

```diff
@@ -226,8 +226,11 @@ type MetricTarget struct {
 type MetricTargetType string
 
 var (
-	UtilizationMetricType  MetricTargetType = "Utilization"
-	ValueMetricType        MetricTargetType = "Value"
+	// UtilizationMetricType is the type of utilization metric
+	UtilizationMetricType MetricTargetType = "Utilization"
+	// ValueMetricType is the type of value metric
+	ValueMetricType MetricTargetType = "Value"
+	// AverageValueMetricType is the type of average value metric
```

Comment:
These aren't accurate… suggest checking the field documentation for where these are used for more accurate descriptions

Reply:
OK, I'll revise it later.

Reply:
I've updated the code, but it still doesn't feel quite right.
Can anyone give me some advice? @feiskyer @yifan-gu @luxas